### ISSUE TITLE
wedge centering

### ISSUE THREAD

**Issue by marcus7070** [CadQuery/cadquery#610]
I think the centering in `Workplane.wedge` is incorrect. I'd like to fix it but just want to make sure I'm not breaking anything, especially given I've never used `wedge` before.

```python
w = cq.Workplane().wedge(10, 10, 10, 4, 4, 6, 6, centered=(True, True, True))
```

![screenshot2021-01-31-203112](https://user-images.githubusercontent.com/50230945/106380616-63625400-6403-11eb-849f-835cef43e646.png)

Note that the bottom left corner is on the origin, which is what I would expect from `centered=(False, False, False)`.

Change centered to `(False, True, True)` and you get:

![screenshot2021-01-31-203359](https://user-images.githubusercontent.com/50230945/106380658-b3411b00-6403-11eb-82dd-3cb12c8c9088.png)

Now the shape doesn't touch the origin at all. The Y and Z directions also behave the same.

Shall I change it such that `centered=(True, True, True)` puts the center of a bounding box around the wedge on the origin?

FYI the last time the `wedge` method was significantly changed was in #228.

**Comment by jmwright** [comment]
My vote would be to fix it so that it works correctly. I don't think wedge gets used very much and that's probably why it hasn't been noticed sooner. Looking at the code, I think it's just an oversight that causes it to not be centered correctly. In #228 I was focused on fixing other things and apparently overlooked the centering problem.

https://github.com/CadQuery/cadquery/blob/3348c18889b37b7489468b4a0f59d73494a67389/cadquery/cq.py#L3532

**Comment by adam-urbanczyk** [comment]
> My vote would be to fix it so that it works correctly.

I agree, looks like an easy fix.

**Comment by adam-urbanczyk** [comment]
Let's fix this and finally release 2.1
